src/Components/SceneList/Internal/SceneDialog.state.ts

```diff
@@ -0,0 +1,104 @@
+import produce from 'immer';
+import {
+    SceneDialogAction,
+    SceneDialogState,
+    SelectionModeOf3DFile,
+    SceneDialogActionType
+} from '../SceneList.types';
+import { IScene } from '../../../Models/Types/Generated/3DScenesConfiguration-v1.0.0';
+
+export const getDefaultSceneDialogState = (
+    scene: IScene | undefined
+): SceneDialogState => ({
+    scene: scene || {
+        id: '',
+        displayName: '',
+        description: '',
+        latitude: undefined,
+        longitude: undefined,
+        elements: [],
+        behaviorIDs: [],
+        assets: []
+    },
+    sceneBlobUrl: scene ? scene?.assets?.[0]?.url || '' : '',
```

Comment:
this doesn't seem quite right. Can we get it to be the nicely formatted value? main on the left, your branch on the right.
![image](https://user-images.githubusercontent.com/57726991/173449130-7e0c9919-f09c-4210-b8c0-30f1ac11d8f8.png)